lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -14,14 +14,21 @@ import { Configuration } from "../config/Configuration";
 import { ServerAuthorizationTokenResponse } from "../server/ServerAuthorizationTokenResponse";
 import { NetworkResponse }  from "../network/NetworkManager";
 import { ScopeSet } from "../request/ScopeSet";
+import { ResponseHandler } from "../response/ResponseHandler";
+import { AuthenticationResult } from "../response/AuthenticationResult";
+import { Serializer } from "../unifiedCache/serialize/Serializer";
 
 /**
  * Oauth2.0 Authorization Code client
  */
 export class AuthorizationCodeClient extends BaseClient {
 
+    // config
+    private clientConfig: Configuration;
+
     constructor(configuration: Configuration) {
         super(configuration);
+        this.clientConfig = configuration;
```

Comment:
This is already in BaseClient, you should be able to access it from here.

Reply:
Yep, changed.